lib/resources/r4/related_person_patch.yaml

```diff
@@ -128,7 +139,47 @@ operations:
       }
     note: >
       <ul>
-        <li>Requires a test operation to be provided for the `Relationship.id` field of the relationship to be removed.</li>
+        <li>Requires a <a href="#test-relationship-id">test operation</a> to be provided for the `Relationship.id` field of the relationship to be removed.</li>
+      </ul>
+
+  - name: replace-relationship-extensions
+    path: /relationship/{index}/extension
+    operation: replace
+    type: string
+    url: https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.relationship
+    description: Replace the `period` and `relation` extensions of the relationship at the given <code>{index}</code>.
+    example: |
+      {
+        "op": "replace",
+        "path": "/relationship/0/extension",
+        "value": [
+          {
+            "valuePeriod": {
+              "start": "2016-12-30T22:26:08.000Z",
+              "end": "2036-12-30T22:26:08.000Z"
+            },
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/period"
+          },
+          {
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v3-RoleCode",
+                  "code": "SIS"
+                }
+              ]
+            },
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/relation"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Requires a <a href="#test-relationship-id">test operation</a> to be provided for the `Relationship.id` of the relationship whose `period` and `relation` is intended to be replaced.</li>
```

Comment:
nit: `... of the relationship whose \`extension\` field is intended to be replaced.`

cc. @alexbeckner1

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/662/commits/300867b2eb8e04f2456f4a36f5c45cc93172f23b#diff-a0c01bda612e4656e850590d65d36528f1caec8a70317c2ae4cb71d8095c9e7dR178

Reply:
+1